docs/authorization-introduction.md

```diff
@@ -0,0 +1,96 @@
+# Authorization Introduction
+
+This document provides a general overview on what are the terms used in the Udaru authorization system and describes the structure and functionality of the authorization application.
+
+The Udaru authorization system has two main functional components:
+- Using the PBAC authorization mechanism for internal use to authorize the system internal parts against itself,
+- Provide PBAC authorization service for external applications.
+
+## Authorization elements
+
+The access control management evolved from the need to mitigate the risk of unauthorized access to data.
+
+Several control models were built to approach the various access control needs: Access Control Lists, Role Based Access Control, Attribute Based Access Control, Policy Based Access Control, Risk Adaptive Access Control.
```

Comment:
i'll rephrase it, my intention is to shortly enum them then provide a quick link to a doc explaining the difference between control models types and their limitations.